daemon/src/maker_cfd.rs

```diff
@@ -421,19 +415,16 @@ impl Actor {
 
         let mut conn = self.db.acquire().await?;
 
-        insert_new_cfd_state_by_order_id(
-            order_id,
-            &CfdState::PendingOpen {
-                common: CfdStateCommon {
-                    transition_timestamp: SystemTime::now(),
-                },
-                dlc: dlc.clone(),
-                attestation: None,
+        let mut cfd = load_cfd_by_order_id(order_id, &mut conn).await?;
+        cfd.state = CfdState::PendingOpen {
```

Comment:
Note: Could be moved inside `handle` towards https://github.com/comit-network/hermes/issues/200 - not sure if it's worth it though, given that we might change the state model to https://github.com/comit-network/hermes/issues/290

On that note: Shall we close https://github.com/comit-network/hermes/issues/200 because https://github.com/comit-network/hermes/issues/290 will likely make it obsolete?

Reply:
Closing #200 sounds good to me.